app/controllers/messages_controller.rb

```diff
@@ -6,12 +6,12 @@ class MessagesController < ApplicationController
   def index
     if params[:view] == 'outbox'
       @page_title = 'Outbox'
-      from_table = current_user.sent_messages.where(visible_outbox: true).order('thread_id, id desc').select('distinct on (thread_id) messages.*')
-      @messages = Message.from(from_table).select('*').order('subquery.id desc').paginate(per_page: 25, page: page)
+      from_table = current_user.sent_messages.where(visible_outbox: true).ordered_by_thread.select('distinct on (thread_id) messages.*')
+      @messages = Message.from(from_table).select('*').ordered_by_subquery.paginate(per_page: 25, page: page)
```

Comment:
Can we remove `ordered_by_subquery`? It's not a Message-specific ordering, so should be done elsewhere if we want to consolidate it (but we probably don't because subquery ordering is fuzzy and complicated)